yaml/kubernetes/security/secrets-in-config-file.yaml

```diff
@@ -0,0 +1,34 @@
+rules:
+  - id: secrets-in-config-file
+    patterns:
+      - pattern: |
+          $KEY: $VALUE
+      - pattern-inside: |
+          data: ...
+      - pattern-inside: |
+          kind: Secret
+          ...
+      - metavariable-regex:
+          metavariable: $VALUE
+          regex: (?i)^[aA-zZ0-9+/]+={0,2}$
+      - metavariable-analysis:
```

Comment:
This analyzer is experimental and requires semgrep 0.83.0 - CI pipelines using `semgrep-agent` cannot currently use this rule

Reply:
Which is OK - just know that it will be a little while (~week?) before this rule will go live :)